models/xero__general_ledger.sql

```diff
@@ -1,3 +1,41 @@
+/*
+in order to generate trancking_category_name as a column we set a dictionary
+dictionary has the following appareance{'tracking_category_id' : [tuple of available id], 'tracking_category_name': [tuple of available names]}
+position match for id and name since the dictionary was generated with a sql statement 
```

Comment:
Not sure I understand this last sentence?

Reply:
I was trying to explain that the first id value matches the first id name, and so on
This is the current dictionary that is generated: 
```
current_dict = {
    'tracking_category_name': ["category_name1", "category_name2", "category_name3"],
    'tracking_category_id' :["id1", "id2", "id3"]
}
```

Instead of this type of dictionary: 
```
possible_dict = {
    "category1": {
        "id": "id1", 
        "name": "category_name1"
    },
    "category2": {
        "id": "id2", 
        "name": "category_name2"
    },
    "category3": {
        "id": "id3", 
        "name": "category_name3"
    },
}
```

Reply:
Ohh, ok. Makes sense, I didn't think of that when I read it. Maybe something like: `Order matters for these arrays, as the N `tracking_category_id` corresponds to the element with the N `tracking_category_name``? Wording still has some room for improvement hah